File: Coding Challenges/DailyChallenge/JavaScript/20260121_MarkdownInlineCodeParser.js
function parseInlineCode(markdown) {
  let tickOn = false;
  let newString = '';

  for(let i = 0; i < markdown.length; i++) {
    let char = markdown[i];

    if(char == '`') {
      newString += `<${tickOn ? '/' : ''}code>`;
      tickOn = !tickOn;
    }
    else {
      newString += char;
    }
  }

  return newString;
}

File: Coding Challenges/Daily Challenge/Python/20260121_MarkdownInlineCodeParser.py
def parse_inline_code(markdown):
    tick_on = False
    new_string = ''

    for char in markdown:
        if char == '`':
            new_string += f'<{"/" if tick_on else ""}code>'
            tick_on = not tick_on
        else:
            new_string += char
            
    return new_string